.vscode/settings.json

```diff
@@ -28,6 +28,8 @@
   "eslint.workingDirectories": [{ "pattern": "packages/*/" }],
   "Lua.runtime.version": "Lua 5.1",
   "Lua.diagnostics.globals": ["vim", "talon", "it", "describe"],
+  "Lua.diagnostics.ignoredFiles": "Disable",
```

Comment:
does that actually work? An example of what is ignored?

Reply:
It works for me fine on vscode in Linux. It ignores the files on the line below:
"Lua.workspace.ignoreDir": ["data/playground/lua/"],

So in that case it ignores data/playground/lua/lua.lua which is the only file in there.

I needed to add both options in order to get it to work.

Reply:
fwiw you should be able to flip between workspaces to trigger a reload instead of having to close vscode entirely. Maybe there is ao even better way to refresh...